docs/poetry.lock

```diff
@@ -842,13 +862,13 @@ dev = ["build", "flake8", "pre-commit", "pytest", "sphinx", "tox"]
 
 [[package]]
 name = "sphinx-substitution-extensions"
-version = "2022.2.16"
+version = "2024.10.17"
 description = "Extensions for Sphinx which allow for substitutions."
 optional = false
 python-versions = "*"
 files = [
-    {file = "Sphinx Substitution Extensions-2022.2.16.tar.gz", hash = "sha256:ff7d05bd00e8b2d7eb8a403b9f317d70411d4e9b6812bf91534a50df22190c75"},
-    {file = "Sphinx_Substitution_Extensions-2022.2.16-py3-none-any.whl", hash = "sha256:5a8ca34dac3984486344e95c36e3ed4766d402a71bdee7390d600f153db9795b"},
+    {file = "Sphinx Substitution Extensions-2024.10.17.tar.gz", hash = "sha256:30fa723bb44afe23adc5187601dcf0c1955dcd1b26f588773b395e852076179d"},
```

Comment:
I have messed with this manually - we should have a CI check that fails on this!

Reply:
well
```
[root@ecbeca56c287 docs]# poetry check --lock
All set!
```